docs/source/features/passes/pytorch.md

```diff
@@ -59,3 +59,24 @@ c. Run QAT training with default training loop.
 
 Check out [this file](https://github.com/microsoft/Olive/blob/main/examples/resnet/user_script.py)
 for an example implementation of `"user_script.py"` and `"create_train_dataloader"`.
+
+## SparseGPT
+`SparseGPT` prunes GPT like models using a pruning method called [SparseGPT](https://arxiv.org/abs/2301.00774). This one-shot pruning method can perform unstructured
```

Comment:
are there any accuracy drop we can refer to between the original GPT model and the pruned model? If yes, we can put it here.

Reply:
The authors measured perplexity on multiple models, sizes and sparsity so I didn't know what results to report here. 
So I linked the original paper for reference. Do you think that's okay?

Reply:
do we have any benchmark data?

Reply:
From Olive? Not yet, I am waiting for access to supported gpus for latency measurements. The only results are those reported in the paper. 
I did get similar perplexity results on a couple of OPT models (2.7B and 4.7B).

Reply:
Updated the doc to say "refer to paper for performance results".

Reply:
do we have plan to run the benchmark for open llama with this technique?

Reply:
Yes, I am planning to run some latency benchmarks for some of the open llama models once I get access to the gpu. I can also add the measure perplexity metrics when adding the latency results. 
Right now, I don't have all of the numbers but wanted to add the pass to main so we can share the example with collaborators.

Reply:
Resolved offline. Will add benchmarks later.